src/Destructurama.ByIgnoring/Destructurama.ByIgnoring.csproj

```diff
@@ -31,8 +31,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Condition="'$(TargetFramework)' == 'netstandard1.0'" Include="Serilog" Version="2.5.0" />
-    <PackageReference Condition="'$(TargetFramework)' == 'netstandard2.0'" Include="Serilog" Version="[2.8.0,3.0.0)" />
+    <PackageReference Include="Serilog" Version="[2.8.0,4.0.0)" />
```

Comment:
There's no need for the upper bound constraint on this anymore; just specifying `2.12.0` would be sufficient, and avoid us hitting the same kind of compatibility issue when Serilog 4 comes around.

Reply:
Good point.  I have pushed a change to PR to remove upper bound.  I successfully ran the build script to verify compilations and tests are good.  I found package version ranges on Microsoft's site helpful ([link](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges)).